src/pipeline_v3/cli/management.py

```
@@ -588,6 +609,103 @@ def _parse_index_type(self, index_type_str: str) -> IndexType:
+            args.processing_options = 'keywords'
+        elif args.processing_options:
+            # New parameter takes precedence
+            pass
+        
+        # Process profile settings if specified
+        if hasattr(args, 'profile') and args.profile:
+            self._apply_profile(args)
+        
+        return args
+    
+    def _apply_profile(self, args):
```

Ruff (0.11.9)

Missing return type annotation for private function `_apply_profile`

Add return type annotation: `None`

(ANN202)